pkg/commands/process/settings/rules/ruby/lang/ssl_verification.yml

```diff
@@ -41,4 +41,6 @@ metadata:
     Coming soon.
     -->
   dsr_id: DSR-2
+  cwe_id:
+    - 319
```

Comment:
It's not cleartext, just not fully verified. I think [CWE-295](https://cwe.mitre.org/data/definitions/295.html) is more appropriate